lighthouse-core/report/html/html-report-assets.js

```diff
@@ -23,6 +23,9 @@ const REPORT_JAVASCRIPT = [
   fs.readFileSync(__dirname + '/renderer/pwa-category-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/report-renderer.js', 'utf8'),
   fs.readFileSync(__dirname + '/renderer/i18n.js', 'utf8'),
+  // // TODO: minified version breaks utf8
+  // fs.readFileSync(require.resolve('pako/dist/pako_deflate.js'), 'utf-8'),
+  fs.readFileSync(__dirname + '/renderer/base64.js', 'utf8'),
```

Comment:
🚲 🏠  felt a bit weird to open the base64 file and see compression streams

encoding
text-encoder
text-encoding


any of those feel better to you?

Reply:
i thought about renaming it, but the end result _is_ base64'd, it's just that the acceptable input is more than just ascii, so...

idk :)

Reply:
File names are about more than just the format of the functions' return types IMO :)

Perhaps I'm alone though...

![image](https://user-images.githubusercontent.com/2301202/119241163-24731e00-bb1a-11eb-8c34-08addf8109b9.png)

Reply:
i also think the name of the file is a bit odd. +1 to `text-encoder`

Reply:
TextEncoder.{encode,decode} also has its oddities, and we can't name the global object `TextEncoder` because that exists :)

Reply:
oh right, globals! lol. `text-encoding` then.

Reply:
Yeah, -1 to `text-encoder` for that reason.

I don't care much what it is though. `base64` is fine to me because we are hopefully not going to grow our encodings collection :) `text-encoding` works too